### PR TITLE
Use imperative mood in docstrings.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
Where class names alone were not grammatical I elaborated with
superclass names to get a valid imperative sentence.

Fixes D401 in pydocstyle and flake8.

Contributes to closing ros2/build_cop#255

Signed-off-by: Steven! Ragnarök <steven@nuclearsandwich.com>

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
CI https://github.com/ros2/build_cop/issues/255#issuecomment-563415682

File: launch/.eggs/README.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
```
Comment (wjwwood) [inline launch/.eggs/README.txt]: Is this file intentional?

Comment (nuclearsandwich) [reply launch/.eggs/README.txt]: No. I'll cut that out.
